### ISSUE TITLE
Slight Issues with Fabric 1.18.2

### ISSUE THREAD

**Issue by DemonSlayer112** [shedaniel/LightOverlay-Fabric#142]
First off, it downloads a cloth mod that had a namechange to UNUSED. Idk if thats the stem of the problems here.
When its installed, it does nothing. F7 does nothing, it has no keybind in vanilla or controlling keybind menu, by no keybind i mean its not even an option.
And at one point it was, i did have it set to F7, but still did nothing on or off.

**Comment by AidenBeau** [comment]
I am having this problem as well

**Comment by JBuschkind** [comment]
Same here

**Comment by Xzatrixs** [comment]
same here

**Comment by jan-Sanku** [comment]
Yes that's most likely the problem. The mod dependency link for Architectury API needs to be updated, as it redirects to the wrong page.
If you're experiencing this issue, try installing architectury from [this](https://www.curseforge.com/minecraft/mc-mods/architectury-api) page, make sure to download the fabric version.

![image](https://user-images.githubusercontent.com/56739341/172006075-cea5b109-821b-44d6-92fd-4945fec79881.png)
